### ISSUE TITLE
Pandas 1.4.0 breaks `make_eval_target_tensors`

### ISSUE THREAD

**Issue by vaiana** [neurallatents/nlb_tools#3]
Hi all,

We noticed that `make_eval_target_tensors` is broken with release of pandas 1.4.0. Here is simple test and the stack trace when installing `nlb_tools` into a clean environment.

```python
from nlb_tools.nwb_interface import NWBDataset
from nlb_tools.make_tensors import make_eval_target_tensors

dataset_path = <path to mc_maze_small dataset>
dataset = NWBDataset(dataset_path)
make_eval_target_tensors(dataset, 'mc_maze_small')
```

```
Traceback (most recent call last):
  File "test_make_eval_target_tensors.py", line 6, in <module>
    make_eval_target_tensors(dataset, 'mc_maze_small')
  File "/home/ubuntu/nlb_tools/nlb_tools/make_tensors.py", line 593, in make_eval_target_tensors
    btrain_dict = make_jagged_array(dataset, [behavior_field], behavior_make_params, train_mask)[0]
  File "/home/ubuntu/nlb_tools/nlb_tools/make_tensors.py", line 748, in make_jagged_array
    data = grouped[trial_ixs[i]][field].to_numpy()
KeyError: 199
```

A workaround is to force `pandas==1.3.4`

**Comment by felixp8** [comment]
Thanks for catching the issue and debugging! I'll fix the dependency requirements ASAP

**Comment by felixp8** [comment]
Should be fixed in 4a9b39e88d1ba3ce80748d39d19691ee84173aed
